HW_1/task_2.py

```diff
@@ -0,0 +1,18 @@
+# Задача 2: Найдите сумму цифр трехзначного числа.
+
+# *Пример:*
+
+# 123 -> 6 (1 + 2 + 3)
+# 100 -> 1 (1 + 0 + 0) |
+
+print("Введите трехзначное число: ")
+number = input()
```

Comment:
Анастасия, не обязательно использовать функцию `print`, чтобы вывести приглашение к вводу. Это сообщение можно поместить напрямую в круглые скобки функции input.
```suggestion
number = input("Введите трехзначное число: ")
```